README.md

```diff
@@ -1,7 +1,44 @@
 # eslint-config-ivy
 
+> ⚠️ **CAUTION**  
+> We now have a published package available! However, since package name has changed from `eslint-config-ivy` to `@houzz/eslint-config-ivy`, we need to maintain two branches for backward compatibility for now.
+
+-   master: For legacy usage (`github:Houzz/eslint-config-ivy#master`).
+-   [release](https://github.com/Houzz/eslint-config-ivy/tree/release): For the new package (`@houzz/eslint-config-ivy`).
```

Comment:
I'm still unsure why we need a `release` branch, but not using `master` branch since this repo is rarely being changed, we can migrate all usages of `github:Houzz/eslint-config-ivy#master` and still have time to update this repo.

Reply:
Because once we update the package name of master branch, I think all CI using the old name will fail right after that. Of course we can update those CI first but it also takes time, therefore what I'd like is to propose a no-dependency solution in this PR.

But I've just reconsidered the solution and figured out that there's a way to keep using master branch and the old package name but also able to publish package with the new package name. Which is to update the package name intermediately during the workflow. I think that'll make thing simplest, I'll update for it next week!

Reply:
Why CI with the old name will fail? `npm i git+https://github.com/Houzz/eslint-config-ivy.git#master` is based on the repo name & branch name, so it just simply get the latest master branch's content

Reply:
ohoh! yes! you're right, I'm so dumb 😆 Just forgot it's not using the package name when installing through https lol. Yeah I just overthinking, will update accordingly then~